docs/docs/ledger/numscript/metadata/index.mdx

```diff
@@ -65,8 +65,22 @@ You can see the `metadata` field with the details of the information we added to
 
 ### What's going on here?
 
-Metadata is arbitrary data that you can attach to a transaction (or an account! But more on that later). This data is typed, and can be a string or a number. You can retrieve (or even add to) this metadata later using the HTTP API, for integration into your own bespoke workflows.
+Metadata is arbitrary data that you can attach to a transaction or an account. This data is typed, and can be a string or a number. You can retrieve (or even add to) this metadata later using the HTTP API, for integration into your own bespoke workflows.
+
+## Attaching account metadata
+
+You can also add metadata to an account during a transaction, like this:
+
+<NumscriptBlock script={`send [COIN 100] (
+  source = @player:leslieknope
+  destination = @centrabank
+)
+set_tx_meta("for_cone", "true")
+set_tx_meta("cone_size", 15)
+set_account_meta(@player:leslieknope, "overdraft", false)
```

Comment:
We don't support boolean type directly in Numscript, you need to use the string type.
```suggestion
set_account_meta(@player:leslieknope, "overdraft", "false")
```